extra/linux/org.alacritty.Alacritty.appdata.xml

```diff
@@ -1,34 +1,45 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2016-2022 Joe Wilm, The Alacritty Project Contributors -->
+<!-- SPDX-License-Identifier: MIT -->
 <component type="desktop-application">
   <id>org.alacritty.Alacritty</id>
-  <!-- Translators: The application name -->
+
+  <developer_name>Christian Duerr</developer_name>
+
   <name>Alacritty</name>
-  <project_license>APACHE-2.0</project_license>
-  <metadata_license>APACHE-2.0</metadata_license>
-  <!-- Translators: The application's summary / tagline -->
-  <summary>A fast, cross-platform, OpenGL terminal emulator</summary>
+
+  <project_license>Apache-2.0</project_license>
+  <metadata_license>MIT</metadata_license>
+
+  <summary>A cross-platform, OpenGL terminal emulator</summary>
   <description>
     <p>
       Alacritty is a modern terminal emulator that comes with sensible defaults,
       but allows for extensive configuration. By integrating with other
       applications, rather than reimplementing their functionality, it manages
       to provide a flexible set of features with high performance.
     </p>
+    <ul>
+      <li>Wayland native</li>
+      <li>X11 native</li>
+      <li>DE agnostic</li>
+    </ul>
   </description>
+
+  <categories>
+    <category>TerminalEmulator</category>
+  </categories>
+
   <screenshots>
     <screenshot type="default">
-      <image>https://user-images.githubusercontent.com/8886672/103264352-5ab0d500-49a2-11eb-8961-02f7da66c855.png</image>
-      <caption>Alacritty - A fast, cross-platform, OpenGL terminal emulator</caption>
+      <image>https://github.com/alacritty/alacritty/blob/0a1e735cf6b13da1dbe5d161d17c8aa6c1692204/extra/promo/alacritty-readme.png</image>
+      <caption>Alacritty with default theme showing neovim</caption>
     </screenshot>
   </screenshots>
-  <keywords>
-    <keyword>terminal emulator</keyword>
-    <keyword>GPU</keyword>
-  </keywords>
+
   <url type="homepage">https://github.com/alacritty/alacritty</url>
   <url type="bugtracker">https://github.com/alacritty/alacritty/issues</url>
-  <update_contact>https://github.com/alacritty/alacritty/blob/master/CONTRIBUTING.md#contact</update_contact>
-  <developer_name>Christian Duerr</developer_name>
+
   <launchable type="desktop-id">Alacritty.desktop</launchable>
+
+  <content_rating type="oars-1.1" />
```

Comment:
According to https://hughsie.github.io/oars/generate.html oars-1.1 only works with gnome software. I don't see why we wouldn't just use 1.0 if it's essentially the same?

Reply:
God I wish we could just leave this off entirely. I get why it exists, I just find it somewhat misplaced. Things like warning people of things like in-app purchases is good, but much of the rest is dubious IMO.

Reply:
> According to https://hughsie.github.io/oars/generate.html oars-1.1 only works with gnome software.

It should also work fine with KDE Discover and other "app stores" such as Flathub.org at this point.

Reply:
Dunno I really want to add graphic violence to Alacritty, I'm just not sure *how* yet. But once I figure that out…

Reply:
Well, you can technically play Quake II in a terminal using [AAlib](https://en.wikipedia.org/wiki/AAlib). :-D

Reply:
> Well, you can technically play Quake II in a terminal using [AAlib](https://en.wikipedia.org/wiki/AAlib). :-D

Luckily, we shouldn't have to worry about the programs people run through alacritty. But I know you were joking.

I was expecting Chris to say something like adding a gun pointing at your foot when you launch on macOS.